frontend/src/components/config/subconfig/ConfigEmailAddress.vue

```diff
@@ -149,245 +149,237 @@
-            this.mailaddressesForDuplicateCheck.has(mail) ||
-            mail.length === 0 ||
-            !this.isEmailValid(mail)
-        );
-    }
+const disableSpeicherButton = computed(() => {
+    const mail: string = editEmailaddress.value.emailAddress;
 
-    get getTitle() {
-        return this.editIndex === -1
-            ? "E-Mail-Adresse anlegen"
-            : "E-Mail-Adresse bearbeiten";
-    }
+    return (
+        isEmpty(mail) ||
+        mailaddressesForDuplicateCheck.value.has(mail) ||
+        mail.length === 0 ||
+        !isEmailValid(mail)
+    );
+});
 
-    /* Rules */
+const getTitle = computed(() => {
+    return editIndex.value === -1
+        ? "E-Mail-Adresse anlegen"
+        : "E-Mail-Adresse bearbeiten";
+});
 
-    /* Prüft, ob ein Wert gesetzt ist. */
-    pflichtfeld(value: any): boolean | string {
-        if (!_.isEmpty(value)) {
-            return true;
-        }
-        return "Hierbei handelt es sich um ein Pflichtfeld. Bitte ausfüllen";
-    }
+/* Rules */
 
-    /* Prüft,ob die Email valide oder nicht leer ist*/
-    isEmailValidOrEmpty(email: string): boolean | string {
-        if (_.isEmpty(email) || this.isEmailValid(email)) {
-            return true;
-        } else {
-            return "Die Email-Adresse ist nicht valide.";
-        }
+/* Prüft, ob ein Wert gesetzt ist. */
+function pflichtfeld(value: string): boolean | string {
+    if (!isEmpty(value)) {
+        return true;
     }
+    return "Hierbei handelt es sich um ein Pflichtfeld. Bitte ausfüllen";
+}
 
-    /* Prüft, ob die EMail bereits hinterlegt ist. */
-    isEmailADuplicate(email: string): boolean | string {
-        if (this.mailaddressesForDuplicateCheck.has(email)) {
-            return "Die Email-Adresse exisitert bereits.";
-        } else {
-            return false;
-        }
+/* Prüft,ob die Email valide oder nicht leer ist*/
+function isEmailValidOrEmpty(email: string): boolean | string {
+    if (isEmpty(email) || isEmailValid(email)) {
+        return true;
+    } else {
+        return "Die Email-Adresse ist nicht valide.";
     }
+}
 
-    /* Prüft, ob eine Email valide ist.  */
-    private isEmailValid(email: string): boolean {
-        const pattern =
-            /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-        return pattern.test(email);
+/* Prüft, ob die EMail bereits hinterlegt ist. */
+function isEmailADuplicate(email: string): boolean | string {
+    if (mailaddressesForDuplicateCheck.value.has(email)) {
+        return "Die Email-Adresse exisitert bereits.";
+    } else {
+        return false;
     }
 }
+
+/* Prüft, ob eine Email valide ist.  */
+function isEmailValid(email: string): boolean {
+    const pattern =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Das Email-Pattern in Utilklasse auslagern. 

Das selbe Pattern wird in `ConfigEmailAdress` und `ConfigDienstleister` verwendet.